### ISSUE TITLE
[BUG] ARIMA handles missing data but tag states otherwise

### ISSUE THREAD

**Issue by ngupta23** [alan-turing-institute/sktime#2204]
The tag in ARIMA model indicates that it does not handle missing data (`'handles-missing-data': False`), but it is able to produce forecasts. Should the tag be changed?

```python
from sktime.datasets import load_airline
y = load_airline()

y_missing = y.copy()
y_missing[2] = np.nan # Just 1 missing value

from sktime.forecasting.arima import ARIMA
forecaster = ARIMA(
    order=(1, 1, 0),
    seasonal_order=(0, 1, 0, 12),
    suppress_warnings=True)
print(forecaster.get_tags())
forecaster.fit(y_missing)
y_pred = forecaster.predict(fh=[1,2,3])
y_pred
```

```python
{'scitype:y': 'univariate', 'ignores-exogeneous-X': False, 'capability:pred_int': True, 'handles-missing-data': False, 'y_inner_mtype': 'pd.Series', 'X_inner_mtype': 'pd.DataFrame', 'requires-fh-in-fit': False, 'X-y-must-have-same-index': True, 'enforce_index_type': None}
1961-01    444.541344
1961-02    418.603889
1961-03    446.815167
Freq: M, dtype: float64
```

**Comment by fkiraly** [comment]
Hm - may be an oversight.

I just wanted to test this systematically, but found that there is no test case with nans in it.
I added one here https://github.com/alan-turing-institute/sktime/pull/2207 to check this tag, in general.

**Comment by ngupta23** [comment]
Thanks @fkiraly . Will you also fix the tag for this model?

**Comment by fkiraly** [comment]
> Will this fix go into the 0.11.0 release?

Probably not - a lot of other estimators are breaking with the nan testing scenario, and I haven´t fully understood whether the problem is with the testing scenario or with the tags...
https://github.com/alan-turing-institute/sktime/pull/2207
